tests/test_structs.py

```diff
@@ -385,6 +385,9 @@ def sampler(s, rng, objs):
     assert isinstance(hash(ground_nsrt), int)
     ground_nsrt2 = nsrt2.ground([cup, plate])
     assert ground_nsrt == ground_nsrt2
+    # Test less than comparison for grounded options
+    assert not ground_nsrt < ground_nsrt2 and not ground_nsrt > ground_nsrt2 # pylint: disable=unneeded-not
```

Comment:
strange pylint complaint, but let's see if we can fix it directly... maybe it wants you to do "x >= y" instead of "x < y". could you instead make another ground nsrt that is slightly different from the original and do:
```
assert ground_nsrt < other_ground_nsrt
assert other_ground_nsrt > ground_nsrt
```
thanks!